src/util/common.js

```diff
@@ -152,9 +152,11 @@ module.exports.getAndroidEnvironmentFile = () => {
 module.exports.makeFileInAndroidMainAssetsFolder = (fileContent, fileName) => {
   try {
     const filePath = path.join(ANDROID_KEYS_DIR_PATH, fileName);
+    fs.removeSync(filePath)
```

Comment:
@talaikis On my side, `outputFileSync` overwrites previous file so no need to remove manually

Reply:
Removed that line